lib/rdf/vocab/dcmitype.rb

```diff
@@ -0,0 +1,143 @@
+# -*- encoding: utf-8 -*-
+# frozen_string_literal: true
+# This file generated automatically using rdf vocabulary format from 
```

Comment:
Do we want these checked into git, or should I add `lib/rdf/vocab/*` to the `.gitignore`?